src/v2/view-builder/views/SignInDeviceView.js

```diff
@@ -0,0 +1,80 @@
+import { loc } from 'okta';
+import { FORMS as RemediationForms } from '../../ion/RemediationConstants';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import BaseFooter from '../internals//BaseFooter';
+import SignInWithDeviceOption from './signin/SignInWithDeviceOption';
+import { getForgotPasswordLink } from '../utils/LinksUtil';
+
+const Body = BaseForm.extend({
+
+  title () {
+    return loc('primaryauth.title', 'login');
+  },
+
+  noButtonBar: true,
+
+  initialize () {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    this.add(
+      SignInWithDeviceOption,
+      { selector: '.o-form-fieldset-container',
+        bubble: false,
+        prepend: true,
+        options: { isRequired: true, form: this }
+      }
+    );
+  },
+});
+
+const Footer = BaseFooter.extend({
```

Comment:
seems identical to `IdentifyView.BaseFooter` ?

Reply:
I will move the footer to shared/IdentifierFooter.js